data/scripts/obtain_item.inc

```diff
@@ -129,10 +136,23 @@ EventScript_PutBattlePyramidItemInBag::
 
 EventScript_FoundTMHM::
 	bufferitemnameplural STR_VAR_1, VAR_0x8004, VAR_0x8005
+    compare VAR_0x8005, 2
+    goto_if_lt EventScript_FoundTMHMMessage
+    buffernumberstring STR_VAR_3, VAR_0x8005
+    message gText_PlayerFoundTMHMs
+    goto EventScript_BufferTMHMsPocket
+    return
```

Comment:
Tabs instead of spaces

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/3942/commits/9350a93cdf6f950be97a2665798693fa05946cd6